server/app/views/v1/volumes/_volume.json.jbuilder

```diff
@@ -5,3 +5,10 @@ json.name volume.name
 json.scope volume.scope
 json.driver volume.driver
 json.driver_opts volume.driver_opts
+json.instances volume.volume_instances.each do |v|
+    json.node v.host_node.name
+    json.name v.name
+end
+json.services volume.services.each do |s|
```

Comment:
`volume.services` is a method call to already query all services which have a `ServiceVolume` that uses this volume:
```
  def services
    self.grid.grid_services.where("service_volumes.volume_id" => self.id)
  end
```
And it doesn't need any relations as least for now.